### DISCUSSION TITLE
Rallly beta

### DISCUSSION THREAD

**Discussion by miguel-negron** [lukevella/rallly#111] [Q&A] [answered]
Hi, 

Does the new version (https://beta.rallly.co/) have a different repository, or it isn't open source?
And anyway, how can we interact with it or talk about it? Here in this repository, in a diferent branch or there is another way?

Cheers.

**Comment by lukevella** [comment, marked as answer]
The beta is in a private repo currently but I will push it to this repo when it's officially released and it will replace the current version. I'm hoping to do that today but but need to push it back a day or two depending on how I get on with my tasks for today.

**Reply by miguel-negron** [reply]
Thanks mate! It's nice to see that very useful protects like this keep going forward. Great work!
